server/storage/schema/migration_test.go

```diff
@@ -51,11 +51,9 @@ func TestNewPlan(t *testing.T) {
 			target:  version.V3_5,
 		},
 		{
-			name:           "Upgrade v3.6 to v3.7 should fail as v3.7 is unknown",
-			current:        version.V3_6,
-			target:         version.V3_7,
-			expectError:    true,
-			expectErrorMsg: `version "3.7.0" is not supported`,
+			name:    "Upgrade v3.6 to v3.7 should work",
+			current: version.V3_6,
+			target:  version.V3_7,
```

Comment:
Maybe we should add a new test for v3.8 to make sure this test is updated when a cut new release?

Reply:
SG, updated & thx